### ISSUE TITLE
Trying to set "render" as property of current state breaks hmr

### ISSUE THREAD

**Issue by yamalight** [gaearon/react-transform-hmr#52]
So, trying to do `this.setState({render: 'val'});` breaks HMR errors wrapper (not 100% sure that's the correct place to report, feel free to close/point me to the right repo).

How to reproduce:
1. Clone [react-transform-boilerplate](https://github.com/gaearon/react-transform-boilerplate)
2. Change method `tick()` in `App.js` to the following: 

``` js
tick() {
    this.setState({
        render: 'whatever',
        counter: this.state.counter + this.props.increment
    });
}
```

You will see the following error in the console:

```
Uncaught TypeError: Cannot read property 'prototype' of undefined
    wrapToCatchErrors @ index.js?9778:32
```

Adjusting the code just a bit makes it work fine, e.g.:

``` js
tick() {
    const s = {
        render: 'whatever',
        counter: this.state.counter + this.props.increment
    };
    this.setState(s);
}
```

It seems that `render` property gets parsed incorrectly in some place along the way.

**Comment by gaearon** [comment]
Thanks for the detailed repro! This has been fixed in `babel-plugin-react-transform@2.0.1` and later via https://github.com/gaearon/babel-plugin-react-transform/pull/86. I would suggest using `babel-plugin-react-transform@latest`.

**Comment by yamalight** [comment]
sweet, thanks!
